packages/http-client-csharp/eng/scripts/Generate.ps1

```diff
@@ -4,11 +4,12 @@ Import-Module "$PSScriptRoot\Generation.psm1" -DisableNameChecking -Force;
 
 $repoRoot = Resolve-Path (Join-Path $PSScriptRoot '..' '..')
 
-Invoke "npm run build"
+Invoke "npm run build:emitter"
+# we don't want to build the entire solution because the test projects might not build until after regeneration
+# generating Microsoft.Generator.CSharp.ClientModel.csproj is enough
+Invoke "dotnet build $repoRoot/generator/Microsoft.Generator.CSharp.ClientModel/src"
```

Comment:
Related comment here. Not sure if our CI/CD runs on linux machines, but previously when I worked in OSX locally and tried to run this script I would run into path not found issues due to how we were concatenating paths.

Reply:
hmm, it looks like regen is only running on windows let me add another job to do the same on linux just to be safe
![image](https://github.com/microsoft/typespec/assets/64171366/92460c2c-a36d-48f4-8232-b6ed3f9b20bf)

Reply:
added so if ci passes we are good
![image](https://github.com/microsoft/typespec/assets/64171366/7f93399a-a774-454f-aae3-2e61c3e3a8bc)